src/SlackChannelServiceProvider.php

```diff
@@ -17,7 +17,7 @@ public function register()
     {
         Notification::resolved(function (ChannelManager $service) {
             $service->extend('slack', function ($app) {
-                return new Channels\SlackWebhookChannel(new HttpClient);
+                return new Channels\SlackWebhookChannel(app(HttpClient::class));
```

Comment:
Use the injected container.

```suggestion
                return new Channels\SlackWebhookChannel($app->make(HttpClient::class));
```